Side Bar.sublime-menu

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "caption": "Lsp: Rename",
```

Comment:
From the sidebar we can rename a file and a folder. 
The name `"Lsp: Rename"` is maybe not the best... If anyone has a suggestion, just say.

There is an inconsistency with naming: `"Lsp: Rename"` and `"LSP: Rename File"`.
To make things a bit challenging to name, we have `"Lsp: Rename"` in the command palette, 
and it is used for symbol renaming.

Reply:
Added a commit to have "LSP: Rename File" and "LSP: Rename Folder"